AGMS-240 - database support in docs (#658)
aerogear/mobile-docs: modules/ROOT/pages/_partials/sync/server-data-access.adoc

= Data Access

> A {sync-service} application will most likely need to access to a database.

Reviewer: ```suggestion
Typically, a {sync-service} application needs to access to a database.
```

> NOTE: Red Hat recommends that you store data in a secure location.

Reviewer: {org-name}
